src/redux/reservations/reservations.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-alert */
+/* eslint-disable default-param-last */
+import axios from 'axios';
+
+const reserveCarAPI = 'https://final-capstone-api.onrender.com/reservations';
+// const reserveCarAPI = 'http://localhost:3000/reservations';
+
+// Actions
+const RESERVE_CAR = 'RESERVE_CAR';
+
+// Initial State
+const initialState = {
+};
+
+// Action Creation
+export const reserveCar = (payload) => ({
+  type: RESERVE_CAR,
+  payload,
+});
+
+// Post Cars
+export const reserveCarAsync = (obj) => async (dispatch) => {
+  const objtwo = {
+    reserve_date: obj.reserve_date.slice(0, 10),
+    car_id: +obj.car_id,
+    user_id: obj.user_id,
+  };
+  const response = await axios.post(reserveCarAPI, objtwo);
+  const data = await response.data;
+  dispatch(reserveCar(data));
+  alert('data');
```

Comment:
- [ ] Your project has 2 alerts.  Here at Microverse, alerts are highly discouraged according to the javascript [requirements](https://github.com/microverseinc/curriculum-final-capstone/blob/main/projects/business_requirements.md#project-requirements):

![image](https://user-images.githubusercontent.com/57421823/212441356-e458334b-ab31-4916-9d2c-10d15b76e339.png)

You can use a modal, or a text that appears on the screen. 👍

Reply:
alert removed from here